include/rdma/fi_log.h

```diff
@@ -35,8 +35,6 @@
 #ifndef FI_LOG_H
 #define FI_LOG_H
 
-#include "config.h"
```

Comment:
> user's configuration may be generated not by autotools, and ENABLE_DEBUG may be omit there (and config.h as well), in this case compiltion will be aborted on user's side

What do you mean by "user's configuration"? This config.h file is always generated by autotools. If this file is there, then it was generated by auto tools and `ENABLE_DEBUG` is defined one way or the other. The `config.h` file is not a general configuration file that a user can define.